### ISSUE TITLE
Each commit today triggers Avast notifications

### ISSUE THREAD

**Issue by kWeb24** [MetaMask/metamask-extension#4331]
Connection with api.infura.io was blocked.
Reason: JS:ScriptSH-inf Trojan
Endpoint: Something with getBlockByNumber from api.infura.io

**Comment by MAtomsS** [comment]
Same here, screenshot of what avast says to me
![image](https://user-images.githubusercontent.com/39503402/40332749-76df6536-5d12-11e8-8868-eb4a53a0b031.png)

**Comment by kWeb24** [comment]
@MAtomsS You've got chrome, i've got firefox so both versions are affected. Probably CI triggers some kind of update and avast freaks out

**Comment by kumavis** [comment]
@kWeb24 in your original title you said "each commit" did you mean "each request" ?

**Comment by kumavis** [comment]
This has been reported to infura and they are reporting it as a false positive to avast

**Comment by kWeb24** [comment]
@kumavis I just noticed that Avast notification pop up right after CI finished. First notification was few seconds after [#4328](https://github.com/MetaMask/metamask-extension/pull/4328) and the same repeats for the rest, #4319 and #4295

**Comment by kumavis** [comment]
@kWeb24 I'm confused - we use "CI" to refer to our CircleCI testing configuration. Are you referring to running the tests locally? or somehow seeing Avast inside CircleCI? or is this for your own CI setup?

**Comment by kumavis** [comment]
MetaMask currently does not have any auto-deploy / continuous deploy system in place, so CI completing does not push updates to clients.

**Comment by kWeb24** [comment]
@kumavis Yep, i mean you're testing configuration with CircleCI. Notifications time was related to finishing tests (as we could see test status directly in repo), that's my theory and could be just a coincidence. So if there's no auto-deploy it's pretty weird or i'm wrong. ;)

**Comment by kumavis** [comment]
I think its unlikely that they are related

**Comment by kumavis** [comment]
Will post an update when we hear from Avast

**Comment by kWeb24** [comment]
Ok, I believe that's false positive so Avast should just confirm that. Have a nice day. ;)
